ERCS/erc-TBA.md

```diff
@@ -0,0 +1,250 @@
+---
+eip: TBA
```

Comment:
```suggestion
eip: 7717
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.